src/pages/recipes/script-result/index.md

```diff
@@ -0,0 +1,54 @@
+# Setting Script Result
+This section contains information on how the result of an UXP script can be set.
+
+## Overview
+InDesign now has the functionality to "set result" of a UXP script. After execution, a script can have its own result which then can be used for debugging purposes on InDesign Server. The result of the script can also be stored in a variable and be used by another script in both InDesign App and InDesign Server. Setting the script result can be accomplished using the script.`setResult()` API, the sections below demonstrate the various use cases. 
+
+## Usage
+The following line sets the result of the UXP script being executed to the string `"Hello World!"`.
+
+```js
+script.setResult("Hello World!");
+```
+---
+## Debugging on InDesign Server
+If we want to debug any variable, we can pass the variable name in the script.setResult() API. This will not only give the variable value but also the type. The following script demonstrates this. 
+
+```js
+let myInDesign = require("indesign");
+let app = myInDesign.app;
+const script = require("uxp").script;
+let testVar = "Hello World!"
+script.setResult(testVar);
+```
+The result after executing this script on InDesign Server is shown on the client side. 
```

Comment:
Whats the client side? Is it InDesign?
Do we need to perform any special steps on the IDS side?

Reply:
Discussed in meeting. Client side refers to the terminal window from where the user executes scripts in IDS.